Bottom: 550aee0dbc7845c62033d7b132057f7388ab2af4
Top:    89e6006df14eb98451d4899acc0456fdf5e16544
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 17:02:51 +0000

Refresh of play-with-quad-edge-data

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 71c8a08..f6ebcdd 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -275,20 +275,9 @@ object3d_from_board_outline (void)
       ct = ct->next;
     }
 
-  /* We know how many edges and vertices we need now...
-   *
-   * let n = npoints
-   * bodies = 1             (FOR NOW - just the first board outline)
-   * vertices = 2n          (n-top, n-bottom)
-   * edges = 3n             (n-top, n-bottom, n-sides)
-   * faces = 2 + n          (1-top, 1-bottom, n-sides)
-   *
-   * holes = 0              (FOR NOW - just the outline, no holes)
-   * holes = ncontours - 1  (LATER)
-   */
-
-  vertices = malloc (sizeof (vertex3d *) * 2 * npoints);
-  edges    = malloc (sizeof (edge_ref  ) * 3 * npoints);
+
+  vertices = malloc (sizeof (vertex3d *) * 2 * npoints); /* (n-bottom, n-top) */
+  edges    = malloc (sizeof (edge_ref  ) * 3 * npoints); /* (n-bottom, n-top, n-sides) */
 
   /* Define the vertices */
   ct = contour;
